faq.qmd

```diff
@@ -0,0 +1,63 @@
+---
+title: "Frequently Asked Questions"
+format:
+  html:
+    toc: true
+---
+
+### Question: Are you part of (Posit, Google, etc)? {#independent}
+
+Answer: No! 
+We are an independent organization.
+
+We are a [fiscally sponsored collective within the Open Collective Foundation](fiscal_sponsorship.qmd).
+We are not affiliated with any other organization.
+
+We have received donations from [Posit](https://posit.co/), and Google mentions us in their [Data Analytics Certificate](https://grow.google/certificates/data-analytics) courses, but we are not part of either organization.
+
+### Question: Are you changing your focus from R to Python? {#focus}
+
+Answer: No! 
+We seek to support *all* data science learners.
+
+We began with R, and our internal tools are written in R.
+However, we know that many data scientists use Python, Julia, and other languages in their day-to-day work.
+We want to help data science learners learn all of the skills they need, regardless of whether those skills involve R, Python, Julia, JavaScript, or no programming language at all.
```

Comment:
```suggestion
We began with R, our internal tools are written in R, and many of our users, book clubs and channels focus on R. 
However, we know that data scientists use many languages in their day-to-day-work, including Python, Julia, SQL.
We want to help data science learners learn all of the skills they need, regardless of whether those skills involve R, Python, Julia, SQL, JavaScript, or no programming language at all.
```

Reply:
Fixed in separate commit. Note that "day-to-day" is an adjective phrase modifying "work", thus "day-to-day work".